release/archives/linux/data-prepper-x64.sh

```diff
@@ -5,6 +5,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if [[ $# -gt 0 ]]
+  then
+    echo
+    echo "Data Prepper expects no command line arguments. Configuration files should be placed in "
+    echo "the application home directory and will be read automatically."
+    echo
+    exit 1
+fi
+
```

Comment:
This is to address this comment: https://github.com/opensearch-project/data-prepper/pull/1737#discussion_r969938840

I'm also thinking about adding back the support for the old command line arguments as I noticed it might still benefits certain use cases. It's easier to switch between different configs from command line arguments than finding the right directories and swapping config files there.

Reply:
Cool. I will add it back in this PR then.

Reply:
Did it in [f0adae3](https://github.com/opensearch-project/data-prepper/pull/1760/commits/f0adae387c780d28f776a3ff02401d42cbc738d8)